conf/defaults.ini

```diff
@@ -369,6 +369,9 @@ content_security_policy_report_only_template = """script-src 'self' 'unsafe-eval
 # Controls if old angular plugins are supported or not. This will be disabled by default in future release
 angular_support_enabled = true
 
+# The CSRF check will be skipped if the login cookie is not present when this value is set to true.
+csrf_skip_check_if_login_cookie_not_present = true
```

Comment:
would be it be possible to turn this into a positive option instead of a negative one?

Examples:

```
csrf_login_only_check
```
or
```
csrf_always_check
```

Reply:
Yes, we can change to `csrf_always` or `csrf_login_only_check`.

Should we default to always checking? (the current behavior doesn't always check)

Reply:
Changed the name to `csrf_always_check` and defaulted it to `false`

Reply:
Hi! Just my 2 cents here: the default behavior should be the same as before, this is a request from one of our external partners, which uses a auth proxy in front of grafana, and therefore the cookie is not set. So, in fact, we don't really want our default/current value to change, but just give them the ability to adapt it.